config/locales/in_person_proofing/fr.yml

```diff
@@ -40,7 +40,7 @@ fr:
         po_search:
           address_search_hint: 'Exemple : 1960 W Chelsea Ave Allentown PA 18104'
           address_search_label: Entrez une adresse pour trouver un bureau de poste près de chez vous.
-          none_found:
+          none_found: Placeholder %{address}
```

Comment:
Best to actually translate this before merging (or mark as draft for now) since this is what a user would see in production.

Reply:
I guess if there's an outstanding story it may be fine since this is behind a FF.